wasp/widgets.py

```diff
@@ -206,10 +211,13 @@ def draw(self):
     def update(self):
         """Update from scrolling indicator."""
         draw = watch.drawable
+        
         if self.up:
-            draw.rleblit(icons.up_arrow, pos=self._pos, fg=0x7bef)
+            draw.rleblit(icons.up_arrow, pos=self._pos,
+                         fg=wasp.system.theme('scroll-indicator'))
```

Comment:
Ideally we grab this once and store it in a local (locals are a bit special in micropython because the go on a stack and don't have names... so they don't consume memory in the same was as other idenfiers).